web/src/lib/utils/asset-utils.ts

```diff
@@ -189,7 +189,7 @@ export function getAssetRatio(asset: AssetResponseDto) {
 	let width = asset.exifInfo?.exifImageWidth || 235;
 	const orientation = Number(asset.exifInfo?.orientation);
 	if (orientation) {
-		if (orientation == 6 || orientation == -90) {
+		if (orientation == 6 || orientation == -90 || orientation == 8) {
```

Comment:
Would the intention of this snippet be clearer?

```suggestion
		if ([-90, 6, 8].includes(orientation)) {
```

Reply:
I'd prefer to leave it as is. IMO, the suggestion seems a bit redundant

Reply:
Personally I think the rotation data should be standardized during exif extraction and then we could reference a single option/value here. Doing it like this means the mobile app and the web and every other consuming client needs to handle all these edge cases and inconsistencies over and over again.